lib/link.js

```diff
@@ -146,7 +146,7 @@ Link.prototype._attachReceived = function(attachFrame) {
   this.emit(Link.Attached, this);
   this._resolveAttachPromises(null, this);
 
-  this._checkCredit();
+  this._checkCredit(/* initial = */ true);
```

Comment:
I don't think so other than object instantiation vs. boolean, but since it's only on attach it's not like this is inner-loop code.